src/NUnitFramework/tests/Attributes/ApartmentAttributeTests.cs

```diff
@@ -47,6 +53,9 @@ public void TestWithRequiresSTARunsInSTA()
         [Test]
 #if THREAD_ABORT
         [Timeout(10000)]
+#endif
+#if PLATFORM_DETECTION && NETCOREAPP2_0
```

Comment:
`NETCOREAPP2_0` implies `PLATFORM_DETECTION`, so I'd prefer the `PLATFORM_DETECTION` check to be removed for all of these. If `NETCOREAPP2_0` ever stops implying `PLATFORM_DETECTION`, build failures are exactly what we want.